lib/matplotlib/axes/_axes.py

```diff
@@ -4934,6 +4941,11 @@ def fill_betweenx(self, y, x1, x2=0, where=None,
         x1 = ma.masked_invalid(self.convert_xunits(x1))
         x2 = ma.masked_invalid(self.convert_xunits(x2))
 
+        for array in [('x', x), ('y1', y1), ('y2', y2)]:
```

Comment:
I think there is a slight mistake here: the arguments are "y", "x1" and "x2".
The documentation and examples don't run.